reana-ui/src/pages/workflowDetails/components/WorkflowLogs.js

```diff
@@ -21,15 +21,26 @@ import { CodeSnippet, TooltipIfTruncated } from "~/components";
 
 import styles from "./WorkflowLogs.module.scss";
 
-export default function WorkflowLogs({ id }) {
-  const dispatch = useDispatch();
-  const loading = useSelector(loadingDetails);
-  const logs = useSelector(getWorkflowLogs(id));
-  const [selectedStep, setSelectedStep] = useState();
+function EngineLogs({ logs }) {
+  return logs ? (
+    <CodeSnippet dollarPrefix={false} classes={styles.logs}>
+      {logs}
+    </CodeSnippet>
+  ) : (
+    <Message
+      icon="info circle"
+      content="There are no workflow engine logs available yet."
```

Comment:
Whlist the workflow is running/queued/pending, it would be better to say something like:

- The workflow engine logs will be available after the workflow run finishes.

And, when there the execution finishes, usually there will be something, also for failed/stopped/etc runs. But if there is still nothing, we can display a generic message:

- There are no workflow engine logs for this execution run.

The motivation is simply to tell users when to expect logs to be visible.  ("yet" is too generic)